wasm/leb128/read_test.go

```diff
@@ -0,0 +1,24 @@
+package leb128
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestReadVarUint(t *testing.T) {
+	n, err := ReadVarUint(bytes.NewReader([]byte{0x80, 0x7f}))
+	if err != nil {
+		t.Fatal(err)
+	}
+	if n != uint32(16256) {
+		t.Fatalf("%d (expected) != %d", n, 16256)
+	}
+
+	sn, err := ReadVarint(bytes.NewReader([]byte{0xFF, 0x7e}))
```

Comment:
that's not testing `ReadVarUint` anymore is it ? :)
just put it into its own `TestReadVarint32` function.